data/migrations/versions/a32e17bfad20_increase_upstreamregistry_s_password_.py

```diff
@@ -0,0 +1,56 @@
+"""increase upstreamregistry's password length (PROJQUAY-7430)
+
+Revision ID: a32e17bfad20
+Revises: 2664723e1b4b
+Create Date: 2024-07-13 20:47:26.649828
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "a32e17bfad20"
+down_revision = "2664723e1b4b"
+
+import sqlalchemy as sa
+
+
+def upgrade(op, tables, tester):
+    with op.batch_alter_table("repomirrorconfig") as batch_op:
```

Comment:
Seeing this error in the mysql tests:
```
All MySQL CHANGE/MODIFY COLUMN operations require the existing type.
FAILED: All MySQL CHANGE/MODIFY COLUMN operations require the existing type.
```
You'll probably need to add the `existing_type` parameter to your alter table statements.